### ISSUE TITLE
Security policies. deleted file is using a space quota

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/ocis#6877]
test on https://0001.schule.owncloud.works/

web 3.1.0-beta.1+16f54d9bb
backend 7.1.0-rc.1

Steps:

- create project space and check quota
- upload `file.log` that doesn't fit security policies
- reload page

Actual:  file was deleted but the quota for that file has been used: 
<img width="397" alt="image" src="https://github.com/owncloud/ocis/assets/84779829/ba348d11-8889-4ede-bf13-4153963f595e">

https://github.com/owncloud/ocis/assets/84779829/ff92af51-d432-4eae-943d-a0b68a0e0516

**Comment by ScharfViktor** [comment]
cc @kobergj

**Comment by kobergj** [comment]
Yes this is a bug in reva. Postprocessing only propagates the etag change and doesn't consider the treesize. 

After next upload treesize should get fixed automatically.

**Comment by micbar** [comment]
> After next upload treesize should get fixed automatically.

I fear that is not the case. We are not calculation a new treesize, we only apply the sizediff.

**Comment by kobergj** [comment]
As mentioned the issue is in reva. https://github.com/cs3org/reva/pull/4081 will fix it.  

NOTE: Spaces that are already affected are NOT repaired - it only fixes the underlying issue.
